File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <ctype.h>

int main()
{
	int a,b,c;
    scanf("%d %d %d",&a,&b,&c);

    if(a + b + c != 180)
    {
        printf("Error \n");
    }
    else if( a == b && b == c && c == a)
    {
        printf("Equilateral \n");
    }
    else if ( a == b || b == c || c == a)
    {
        printf("Isosceles \n");
    }
    else{
        printf("Scalene");
    }

	return 0;
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin', 'utf8').trim().split('\n');

const [a, b, c] = input.map(Number);

if ((a+b+c) === 180 && a === b && b === c) {
    console.log("Equilateral");
}else if( (a+b+c) == 180 && (a === b || b === c || a === c)){
    console.log("Isosceles");
}else if((a+b+c) == 180){
    console.log("Scalene");
}else{
    console.log("Error");
}